app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -0,0 +1,72 @@
+class TreeBuilderAlertProfileObj < TreeBuilder
+  def initialize(name, type, sandbox, build = true, assign = nil)
+    @assign = {
```

Comment:
Still wondering if we need all that.. since the tree only looks at `@assign[:new][:assign_to]`, `@assign[:new][:objects]` and `@assign[:new][:cat]`, maybe you could just pass in these tree, so that we don't rely on the structure of `@assign` in this tree.